src/sources/metadatasourcetaglib.cpp

```diff
@@ -757,6 +787,12 @@ MetadataSourceTagLib::exportTrackMetadata(
             << "with type" << m_fileType;
 
     SafelyWritableFile safelyWritableFile(m_fileName, kExportTrackMetadataIntoTemporaryFile);
+    if (safelyWritableFile.fileName().isEmpty()) {
+        kLogger.warning()
+                << "Unable to export track metadata into file"
```

Comment:
Please make this message more detailed. Add "- no target file name supplied, check disk free space." or similar.

Reply:
The actual reason has already been logged by SafelyWritableFile that knows more details, i.e. the error code received from the last file operation. This log message in the outer scope is redundant.

But checking for an empty file name is in fact confusing and reveals implementation details. I will change this.